pkgs/servers/nextcloud/packages/nextcloud-apps.json

```diff
@@ -20,6 +20,7 @@
 , "news": "agpl3Plus"
 , "notes": "agpl3Plus"
 , "notify_push": "agpl3Plus"
+, "oidc_login": "agpl30only"
```

Comment:
```suggestion
, "oidc_login": "agpl3Only"
```

Reply:
should i do another pull request to make the commits prettier?  instead of adding a commit just for a typo?

Reply:
I'd just fix the error and squash afterwards

Reply:
done